src/main/java/graphql/nadel/engine/NadelExecutionStrategy.java

```diff
@@ -340,4 +348,13 @@ private ExecutionContext buildServiceExecutionContext(ExecutionContext execution
         );
     }
 
+    private String buildOperationName(Service service) {
```

Comment:
question is a bit: should we also incorporate the original operation name of the overall query if it is exists?

Reply:
good idea